### PR TITLE
4 books + new section.

### PR DESCRIPTION
**Pull request by dapoadedire**
## Description

I added 4 books and one new section.

One ReactJS book, One HTML/CSS book, and 2 Other books.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -42,6 +42,7 @@ Below are high-level lists of all the resources contained in the repo so far, th
 - [React](/books/dev-books/React/react.md)
 - [TypeScript](/books/dev-books/TypeScript/typescript.md)
 - [Vuejs](/books/dev-books/Vue/vue.md)
+- [Others](/books/dev-books/Others/others.md)
 
 #### Design Books
```
Comment (binadiegha) [inline README.md:47]: let's have a design folder instead. outside of dev book... I'll set this up

File: books/dev-books/HTML and CSS/htmlcss.md [hunk 2]
```diff
@@ -10,25 +10,33 @@ Please kindly follow this structure when you are contributing
...
-- [Mordern CSS with Tailwind](https://drive.google.com/file/d/1L8GZdQUdbBHYW88fgOhAN1pfz2wgWp2e/view?usp=share_link)
+
+- [Modern CSS with Tailwind](https://drive.google.com/file/d/1L8GZdQUdbBHYW88fgOhAN1pfz2wgWp2e/view?usp=share_link)
+
+- [Master the Skill of Debugging CSS](https://drive.google.com/file/d/1cS5q-2GSTBI-ojsush7bIuuHHfGQX_Ok/view?usp=share_link)
 
 ### R
-- [Responsive web Design with HTML and CSS by Ben Frian](https://drive.google.com/file/d/1LBmHA6enY7eLRuh83mXi-ANvT6fPjSK2/view?usp=share_link)
 
+- [Responsive web Design with HTML and CSS by Ben Frian](https://drive.google.com/file/d/1LBmHA6enY7eLRuh83mXi-ANvT6fPjSK2/view?usp=share_link)
```
Comment (binadiegha) [inline books/dev-books/HTML and CSS/htmlcss.md:42]: LGTM ✅

**Review by binadiegha — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (dapoadedire) [reply README.md:47]: All right, that's cool.

**Comment by EOEboh** [comment]
@all-contributors please add @dapoadedire for code

**Pull request merged**